react/features/toolbox/components/web/ProfileButton.js

```diff
@@ -43,13 +48,15 @@ class ProfileButton extends AbstractButton<Props, *> {
      * Retrieves the label.
      */
     get label() {
-        const { _localParticipant } = this.props;
+        const {
+            _defaultLocalDisplayName,
+            _localParticipant } = this.props;
         let displayName;
 
         if (_localParticipant && _localParticipant.name) {
```

Comment:
maybe you can simplify this condition while you're here? `_localParticipant?.name`